player.py

```diff
@@ -0,0 +1,25 @@
+class Player:
+    def __init__(self, name, health):
+        self.name = name
+        self.health = health
+        self.inventory = []
+
+    def pick_up_item(self, item):
+        print("You picked up a", item.name)
+        self.inventory.append(item)
+
+    def give_item(self, item, character):
+        if item in self.inventory:
+            character.inventory.append(item)
+            self.inventory.remove(item)
+
+# cause is what did the damage. damage is how much damage was done.
+def take_damage(self, cause, damage):
```

Comment:
You could put this function in the player class too! I think it might make more sense, that way you don't need to import it separately. Unless there is some reason you don't want it in the player class?